### PR TITLE
Jenkins Cron job - Run Live E2Es and Local tests in parallel

### PR DESCRIPTION
**Pull request by sareh**
No ticket

**Overall change:** Ensures that E2E test stages run on the Cron job are run in parallel

**Before**: Install & Setup, Live E2Es, Local Prod Tests

Screenshot of stages

**After**: Live E2Es & Local Prod Tests run in parallel:

< image to be added>

**Code changes:**

- Moves two stages into a `parallel` block 

**Testing notes**

- TBC

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)~ N/A
- [x] This PR requires manual testing

### CODE REVIEW

File: Jenkinsfile-e2e [hunk 1]
```diff
@@ -43,52 +43,57 @@ node {
       cleanWs() // Clean the workspace
       checkout scm // Checkout from git
       docker.image("${nodeImage}").inside('--ipc host') {
+        parameters {
```
Comment (DemonusPC) [inline Jenkinsfile-e2e:46]: Not used anywhere. You can remove it

**Review by DemonusPC — APPROVED** [review]
Looks good except that parameter which doesnt need to be added

File: Jenkinsfile-e2e (hunk 1 above)
Comment (sareh) [reply Jenkinsfile-e2e:46]: I added this to test the changes to this branch on Jenkins.

Comment (DemonusPC) [reply Jenkinsfile-e2e:46]: nvm then

**Pull request closed without merging**